scripts/library.sh

```diff
@@ -36,6 +36,20 @@ readonly IS_PROW
 readonly REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
 readonly REPO_NAME="$(basename ${REPO_ROOT_DIR})"
 
+# Useful flags about the current OS
+IS_LINUX=0
+IS_OSX=0
+IS_WINDOWS=0
+case "${OSTYPE}" in
```

Comment:
What would the value of `${OSTYPE}` be for a linux container running on OSX?

Reply:
I guess it would be Linux, because it's bash who defines OSTYPE.

Reply:
should we document it here or would it be the caller's job to figure this out?

Reply:
This is a corner case and I can't imagine such situation in the present state of test-infra. If we ever get to this situation, I'm sure way more issues will arise when running these scripts (assuming they will be even being used then) than realizing that for some reason bash wrongly sets OSTYPE.

Reply:
make sense